.github/workflows/registry.yml

```diff
@@ -36,6 +36,14 @@ jobs:
         with:
           compose-file: "./registry/docker-compose.yml"
       - run: rustup update
+      # run registry locally to permit API tests
+      - name: "Run Registry in background"
```

Comment:
We definitely should not do this imho!

We can run the registry virtually through cargo test :) 

If you need help with this we can pair here 😊